### PR TITLE
Update : Twitter Icon

### PR DESCRIPTION
**Pull request by 4ndu-7h4k**
Updated Twitter icon

Dark Theme
image

Light Theme
image

### CODE REVIEW

**Review by kiaking — COMMENTED** [review]
Maybe we might wanna change the file name to `VTIconX.vue` 👀

**Comment by 4ndu-7h4k** [comment]
> Maybe we might wanna change the file name to `VTIconX.vue` 👀

How about VTIconTwitterX.vue ?

**Comment by kiaking** [comment]
> How about VTIconTwitterX.vue?

Why? It is "X" isn't it? 👀

**Comment by kiaking** [comment]
Looking good to me 👍 

@yyx990803 @bencodezen @NataliaTepluhina
When do you think is a good timing to ally this change...? 🤔

**Comment by Jinjiang** [comment]
@kiaking I think we can merge this for now. And then we can further publish it. And then people in the docs project can choose a proper time to apply it.

Let me will check the Netlify checks failure first.

File: src/core/components/VTSocialLink.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@ const icons = {
   github: VTIconGitHub,
   linkedin: VTIconLinkedIn,
   slack: VTIconSlack,
-  twitter: VTIconTwitter,
+  twitter: VTIconX,
```
Comment (Jinjiang) [inline src/core/components/VTSocialLink.vue:25]: For better flexibility to end users, I might prefer having both twitter and x props here. The twitter is still to the blue bird icon. The x is to the new one. So people can feel free to choose what they would like to use. What do you think?

Comment (Jinjiang) [reply src/core/components/VTSocialLink.vue:25]: That also means we'd better keep the file `src/core/components/icons/VTIconTwitter.vue`.

Comment (kiaking) [reply src/core/components/VTSocialLink.vue:25]: Good point. @4ndu-7h4k Would you mind fixing this?

Comment (4ndu-7h4k) [reply src/core/components/VTSocialLink.vue:25]: Sure @kiaking

**Review by Jinjiang — APPROVED** [review]

**Pull request merged**